drivers/usb/device/Kconfig

```diff
@@ -21,6 +21,21 @@ config USB_DW_IRQ_PRI
 	help
 	USB interrupt priority.
 
+config USB_DW_DMA
+	bool "Enable Designware USB Controller Buffer DMA mode"
+	default y
+	help
+	Use buffer DMA instead of FIFO mode for I/O to the controller.
+	This will reduce CPU overhead, but (at least right now)
+	requires a 64 byte per-endpoint buffer be allocated out/into
+	which the transferred data must be copied.  The USB framework
```

Comment:
The same with Kinetis USBFSOTG #542 , I've made the buffers configurable, but I'm not that happy. Maybe we can find better solution later.